addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as ts from 'typescript';
+import * as glob from 'glob';
+import * as path from 'path';
+const Promise = require('ember-cli/lib/ext/promise');
+
+/**
+ * Declare an Interface where key is a string and value is either a string or a number.
+ * Use for storing a string literal, start position and end posittion of ImportClause node kinds.
+ */
+interface ModuleImport {
+  specifier: string;
+  pos: number;
+  end: number;
+};
+
+export interface ModuleMap {
+  [key: string]: ModuleImport;
```

Comment:
Since this interface is exported, `ModuleImport` should be too or Typescript will complain.